in_toto/hash.go

```diff
@@ -0,0 +1,79 @@
+package in_toto
+
+import (
+	"crypto/md5"
+	"crypto/sha256"
+	"crypto/sha512"
+	"fmt"
+)
+
+/*
+Hash is an interface which contains a generic compute method.
+This method is implemented by several functions to return the
+hash for a given data.
+
+To add hash functions which are currently not supported one
+has to create a struct, write it's defination, add it's
```

Comment:
I notice that the typo "defination" in the codebase in a couple of places. Could you correct it please?